agent/src/main.rs

```diff
@@ -42,15 +42,15 @@ async fn main() -> Result<()> {
     env_logger::init();
 
     // TODO: metrics sink initialization based on env vars or config
-    let addr: SocketAddr = "0.0.0.0:9000".parse().unwrap();
+    let metrics_port = std::env::var("METRICS_PORT").unwrap_or_else(|_| "9000".to_string());
```

Comment:
This reminds me about something Johannes brought up the other day; all of our port bindings in serval-mesh are for IPv4 (`0.0.0.0:{port}`) rather than IPv6 (`::{port}`, I believe, although I have not tried it).

Created https://github.com/serval/serval-mesh/issues/50 to track it.